diagnostic reports/README.md

```diff
@@ -0,0 +1,374 @@
+and `CompoundStatement / code [@code]` equals `"16488004"` (laboratory reporting).  
+
+| Mapped to (JSON FHIR DiagnosticReport field) | Mapped from (XML HL7 / other source)                                                                                                                                                                                                                                                                         |
+|----------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| id                                           | `CompoundStatement / id[0] [@root]`                                                                                                                                                                                                                                                                          |
+| meta.profile\[0]                             | fixed value = `"https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-DiagnosticReport-1"`                                                                                                                                                                                                            |
+| identifier\[0].system                        | `"https://PSSAdaptor/{{odsCode}}` where `{{odsCode}}` is the ODS Code of the losing practice                                                                                                                                                                                                                 |
+| identifier\[0].value                         | `CompoundStatement / id[0] [@root]`                                                                                                                                                                                                                                                                          |
+| identifier\[1].system                        | fixed value = `"2.16.840.1.113883.2.1.4.5.5"` <sup>1</sup>                                                                                                                                                                                                                                                   |
+| identifier\[1].value                         | `CompoundStatement / id[1] [@root]` <sup>1</sup>                                                                                                                                                                                                                                                             |
```

Comment:
Appears to be looking at the `extension` attribute?

Reply:
updated, thanks